### PR TITLE
Cleanup Pages & Create Superblocks Wiki Page

### PR DESCRIPTION
**Pull request by RoboticMind**
* Cleans up a bunch of pages
  *  Updates the warning message on the solo guide since the 180 days thing is not true anymore
  * Add new nodes in addnode page 
  * Clarify information about side staking in the config-file page
  * Typos, capitalization, awkward wording, outdated info, etc.
* Creates a wiki page about superblocks and add mentions of it  
* Update staking page 
  * Mention side staking
  *  Mention icons that you will see when you stake

### CODE REVIEW

**Comment by RoboticMind** [comment]
@barton2526 whenever you get the chance, can you merge this if it looks good?

**Comment by barton2526** [comment]
Additional: Removes dead metadata (https://github.com/gridcoin-community/Gridcoin-Site/pull/284/commits/370dfbd06c64e5017046dedb061ff37aa4fd43b3)

File: wiki/faq.md [hunk 1]
```diff
@@ -152,10 +151,9 @@ Yes, there are currently 2 operational pools:
 ---
 ### I am crunching for the pool but the wallet still shows INVESTOR and 0 magnitude. Is that ok?
 
-That is perfectly normal when pool-mining you do work for the pool's
-account and as a reward get transactions from the pool for the work you
-did. Because of this you do not work for your own CPID and have no
-magnitude.
+On the beacon menu, make sure to click on "Pool" instead of investor or solo. Click
```
Comment (barton2526) [inline wiki/faq.md:154]: aren't we removing "investor" verbiage?

File: wiki/faq.md [hunk 2]
```diff
@@ -425,13 +426,14 @@ Just try to just wait it out the wallet should automatically fix itself.
...
+2.  In your Gridcoin data folder, delete the accrual folder, the txleveldb folder, 
+and blk00*.dat (with * being any number).
+3.  Restart the wallet and wait for it to sync. If you don't want to wait you can also use the snapshot
+ which you can use by clicking on File -> Snapshot or using the `-snapshotdownload` [command line argument](cmd-options "wikilink") 
 
 You can also try running the wallet as an administrator.
 
-There is also an [official
+There is also an [
 thread](https://cryptocointalk.com/topic/20303-gridcoin-proof-of-research-connection-sync-problem-thread/)
```
Comment (barton2526) [inline wiki/faq.md:437]: this needs to be archived or deleted, as it is no longer accessible

File: wiki/faq.md [hunk 3]
```diff
@@ -502,21 +504,30 @@ missed the Burn deadline, your coins are now in the possession of the
...
+sending the transaction after that. If you don't have it locked at all, you can 
+ignore this.
+
+If you see `Error: the transaction was rejected` or you see 
+an error saying "not enough fees", you may have too many small UTXOs (what's
+leftover from a transaction you received) to send a transaction. There is a
+maximums transaction size and so if a transaction tries to include to many UTXOs, 
+it can be impossible to send. To fix this, you can try consolidating them using the
+`consolidateunspent` [rpc](rpc "wikilink") command. For each of your addresses,
+try running `consolidateunspent <address> 50`. After this finishes, try sending
```
Comment (barton2526) [inline wiki/faq.md:531]: nit: I don't believe the 50 is necessary because that is the default

File: wiki/superblock.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+# How Superblocks Get Made 
+
+1. A previous superblock is made and a 24 hour clock starts for the next superblock to be due[^1]
+2. Stats are independently gathered by the [scrapers](scrapers "wikilink") from the BOINC projects
+3. Those stats are sent out to each wallet which compare them and check that they match. 
+If at least 60% (rounded up) of the scrapers exactly match[^2], a superblock can be made (when it's due)
+4. Someone stakes and they include the new data from the scrapers
+
+Note that this is just a brief overview and there are many security and other mechanisms left out in the description
```
Comment (barton2526) [inline wiki/superblock.md:27]: "there are many security and other mechanisms" sounds awkward

File: wiki/faq.md (hunk 1 above)
Comment (RoboticMind) [reply wiki/faq.md:154]: I was referring to the "Investor" button in the wallet, but yeah it's not needed. Simplified it to just mention the pool button

File: wiki/faq.md (hunk 2 above)
Comment (RoboticMind) [reply wiki/faq.md]: removed that

File: wiki/superblock.md (hunk 4 above)
Comment (RoboticMind) [reply wiki/superblock.md]: Change it to "Note that this is just a brief overview and there are many security mechanisms and other parts left out in the description"

**Pull request merged**